app/controllers/index.js

```diff
@@ -1,32 +1,60 @@
 import Controller from '@ember/controller';
-import { computed } from '@ember/object';
-import { equal } from '@ember/object/computed';
 import { all } from 'rsvp';
 
-import moment from 'moment';
-
 export default Controller.extend({
- daysUntilPublishing: computed(function() {
-   const today = new Date();
-   return moment(today).day();
- }),
- isPublishingDay: equal('daysUntilPublishing', 5),
- isDayBeforePublishing: equal('daysUntilPublishing', 4),
-  conListUniq: null,
+  contributorsList: '',
+
   actions: {
     async getContributors() {
-      let model = this.get('model');
-      let mergedPulls = model.mergedPulls;
-      let users = await all(mergedPulls.map((pull) => pull.get('user')));
-      let userLinks = users
-        .uniq()
-        .reject((user) => user.get('login').includes('[bot]'))
-        .map((user) => {
-        return `<a href="${user.get('htmlUrl')}" target="gh-user">@${user.get('login')}</a>`;
-      }).join(', ');
-
-      document.querySelector('.fetch-contributors').click();
```

Comment:
I noticed that this line copies the contributors list into the system so that we can paste immediately after.

Unless someone saw this code, I don't think they would know that they can paste without clicking the Copy button on the website. By removing this side effect, we can also reduce the responsibility of `getContributors`.